README.md

```diff
@@ -1,69 +1,73 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Space Traveller's Hub
 
-> One paragraph statement about the project.
+> In this project we created a single page application using react-redux
 
-![screenshot](./app_screenshot.png)
+![screenshot](./images/image.png)
 
-Additional description about the project and its features.
+On this website you can join missions and reserve rockets
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- React and Redux
+- Redux-thunk, Redux-logger, axios.
+- Javascript
 
-## Live Demo
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
 
-### Usage
+## Prerequisites
+- A working Laptop or Desktop
+## Setup
+- Install code editor
+- Install Git bash
+- Create react app
+- Install react-redux
 
-### Run tests
+## Install
 
-### Deployment
+clone [Bookstore](https://github.com/Emmy-github-webdev/space-traveller)
```

Comment:
- [ ] Kindly, work on the readme file, this is a space-traveler project, not a bookstore.